### DISCUSSION TITLE
Domain bought from vercel not appearing in dashboard

### DISCUSSION THREAD

**Discussion by Bijig0** [vercel/community#6490] [Help]
### Summary

I've just bought a domain from Vercel, (have the receipt sent to me), but it's not showing up on my dashboard? When I bought domains previously on Squarespace domains or google domains it was instant, so just wondering what's up here?

### Steps to Reproduce

.

**Comment by versecafe** [comment]
@Bijig0 where are you looking for the domain inside of Vercel? are you checking `vercel.com/[team]/~/domains`

**Reply by Bijig0** [reply]
Yes I am. In the vercel dashboard, I select Domains (the URL appears as shown) but my purchased domain doesn't appear

**Reply by amyegan** [reply]
Do you see the domain on your personal account domain list at [vercel.com/account/domains](https://vercel.com/account/domains)?

**Reply by Bijig0** [reply]
Ah interesting yeah it shows up there but still not on dashboard -> domains though. But now when I try connecting my site to the domain it works! So that's cool haha

**Reply by versecafe** [reply]
@amyegan is this maybe due to the move to give every user a hobby team with a domain that was owned prior to the transition?

**Reply by amyegan** [reply]
Yeah, this seems related to the new hobby team update. The domain can be moved from the personal account domains list (`vercel.com/account/domains`) to the new hobby team to make it appear on dashboard -> domains for the new team (`vercel.com/[team]/~/domains`)

You can find more info about the that in https://github.com/vercel/community/discussions/6298
